target/linux/generic/pending-5.4/834-ledtrig-libata.patch

```diff
@@ -122,28 +122,34 @@ Signed-off-by: Daniel Golle <daniel@makrotopia.org>
  	/* Create associated sysfs transport objects  */
  	for (i = 0; i < host->n_ports; i++) {
  		rc = ata_tport_add(host->dev,host->ports[i]);
---- a/include/linux/libata.h
-+++ b/include/linux/libata.h
+--- a/include/linux/libata.h	2020-06-24 11:50:53.000000000 -0400
++++ b/include/linux/libata.h	2020-06-24 17:35:15.642949054 -0400
 @@ -22,6 +22,9 @@
  #include <linux/acpi.h>
  #include <linux/cdrom.h>
  #include <linux/sched.h>
 +#ifdef CONFIG_ATA_LEDS
 +#include <linux/leds.h>
 +#endif
+ #include <linux/async.h>
```

Comment:
Okay, now I see that this line is required, but other lines seems trivial or duplicate.
I would suggest something like this:
```diff
--- a/target/linux/generic/pending-5.4/834-ledtrig-libata.patch
+++ b/target/linux/generic/pending-5.4/834-ledtrig-libata.patch
@@ -125,9 +125,9 @@ Signed-off-by: Daniel Golle <daniel@makrotopia.org>
 --- a/include/linux/libata.h
 +++ b/include/linux/libata.h
 @@ -22,6 +22,9 @@
- #include <linux/acpi.h>
  #include <linux/cdrom.h>
  #include <linux/sched.h>
+ #include <linux/async.h>
 +#ifdef CONFIG_ATA_LEDS
 +#include <linux/leds.h>
 +#endif
```